src/App.tsx

```diff
@@ -1,26 +1,296 @@
-/* eslint-disable max-len */
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useRef, useState } from 'react';
 import { UserWarning } from './UserWarning';
-
-const USER_ID = 0;
+import { getTodos, USER_ID } from './api/todos';
+import { Todo } from './types/Todo';
+import * as Methods from './api/todos';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [error, setError] = useState<string | null>(null);
+  const [statusFilter, setStatusFilter] = useState('All');
+  const [newTodoTitle, setNewTodoTitle] = useState('');
+  const [isInputDisabled, setIsInputDisabled] = useState(false);
+  const [counter, setCounter] = useState<number>(0);
+  const [loadingTodoId, setLoadingTodoId] = useState<number | null>(null);
+
+  const newTodoInputRef = useRef<HTMLInputElement>(null);
+
+  function updateCounter(arrTodo: Todo[]) {
+    const activeTodosCount = arrTodo.filter(todo => !todo.completed).length;
+
+    setCounter(activeTodosCount);
+  }
+
+  useEffect(() => {
+    if (newTodoInputRef.current) {
+      newTodoInputRef.current.focus();
+    }
+  }, [todos]);
+
+  useEffect(() => {
+    const fetchTodos = async () => {
+      try {
+        const fetchedTodos = await getTodos();
+
+        setTodos(fetchedTodos);
+        updateCounter(fetchedTodos);
+      } catch {
+        setError('Unable to load todos');
+      }
+    };
+
+    fetchTodos();
+  }, []);
+
+  useEffect(() => {
+    if (error) {
+      const timer = setTimeout(() => {
+        setError(null);
+      }, 3000);
+
+      return () => clearTimeout(timer);
+    }
+
+    return undefined;
+  }, [error]);
+
+  function deleteTodo(todoId: number) {
```

Comment:
It is important to follow the same way of creating functions everywhere, fix it everywhere
```suggestion
  const deleteTodo = (todoId: number) => {
```